project-base/storefront/components/Blocks/Product/ProductsList/ProductListItem.tsx

```diff
@@ -118,7 +118,7 @@ export const ProductListItem = forwardRef<HTMLLIElement, ProductItemProps>(
                         />
                     )}
 
-                    {visibleItemsConfig.storeAvailability && (
+                    {visibleItemsConfig.storeAvailability && !product.isInquiryType && (
```

Comment:
Just slightly 🤣
It was a deliberate choice. I can imagine few scenarios where the stock may be useful even for the inquiry products. This gives a freedom to implement these. 
For example we may wan't to distinguish products that are available, but have to be for example engraved or assembled and the products that have to be ordered from the supplier.
And yes it would probably be unnecessarily complex in the backend code.